src/docs/release/breaking-changes/template.md

```diff
@@ -19,49 +25,71 @@ a breaking change?"]
 
 ## Description of change
 
-[A dry technical description of the actual change,
-with code samples.]
-
+{% comment %}
+A dry technical description of the actual change,
+with code samples.
+{% endcomment %}
 
 ## Migration guide
 
-[A description of how to make the change.
+{% comment %}
+A description of how to make the change.
 If a migration tool is available,
 discuss it here. Even if there is a tool,
-a description of how to make the change manually must be provided.]
+a description of how to make the change manually must be provided.
+{% endcomment %}
 
-### Before
+Code before migration:
 
+<!-- skip -->
 ```dart
-[Example of code before the change.]
+// Example of code before the change.
 ```
 
-### After
+Code after migration:
 
+<!-- skip -->
 ```dart
-[Example of code after the change.]
+// Example of code after the change.
 ```
 
 
 ## Timeline
 
-[Details regarding when the change was made, in terms of a version number.
-If there is a deprecation window, the date to which we guarantee to maintain the old API.]
+{% comment %}
+Details regarding when the change was made, in terms of a version number.
+If there is a deprecation window,
+the date to which we guarantee to maintain the old API.
+{% endcomment %}
 
-[This text should be backwards looking. People are reading this months from now, not today.
-Don't say "in a month" or talk about your plan to do something next week. You've done it,
-they're looking back to figure out how to migrate their code.]
+{% comment %}
+This text should be backwards looking, so write in past tense,
+not future tense. People are reading this months from now, not today.
+Don't say "in a month" or talk about your plan to do something next week.
+You've done it, they're looking back to figure out how to migrate their code.
+{% endcomment %}
```

Comment:
one thing i noticed in all the guides so far is that they seem a bit anemic in this section, we should probably work to be more detailed here going forwards. Often people will be looking at these a year from now and they'll want to know "when did this change, what version can i go back to to prepare, is there a deprecation window", etc. I think in the future we shouldn't accept the guides until this section can be filled in; we shouldn't publish guides with TBA in their timeline.

Reply:
@Hixie, I noticed that too and I completely agree. I did what I could to improve those sections, but they are definitely not consistent. I will think about updating this section and will definitely be more strict in future reviews.